plugins/node/opentelemetry-instrumentation-express/test/ignore-all.test.ts

```diff
@@ -127,14 +127,14 @@ describe('ExpressInstrumentation', () => {
       );
     });
 
-    it('rpcMetadata.route should be modified to /todo/:id', async () => {
+    it('rpcMetadata.route should be undefined', async () => {
       assert.strictEqual(memoryExporter.getFinishedSpans().length, 0);
       await context.with(
         trace.setSpan(context.active(), rootSpan),
         async () => {
           await httpRequest.get(`http://localhost:${port}/toto/tata`);
           rootSpan.end();
-          assert.strictEqual(rpcMetadata.route, '/toto/:id');
+          assert.strictEqual(rpcMetadata.route, undefined);
```

Comment:
as all the layer is ignore, we don't have anywhere to update rpcMetadata.route

Reply:
hmm, maybe my hint to move this down after the ignore check was not the best idea.
Route detection and span creation are actual two distinct topics so maybe better to detect route independent of layer span creation.
For irrelevant requests (e.g. health) the ignore must anyway start in http layer.

Reply:
Then should we also remove this logic here?

https://github.com/open-telemetry/opentelemetry-js-contrib/blob/8fefd23877a847f0906cf9c582afef9dd55d395b/plugins/node/opentelemetry-instrumentation-express/src/instrumentation.ts#L200

When the layer is ignore the route layer also pop.